subsys/shell/shell_dummy.c

```diff
@@ -55,13 +57,21 @@ static int write(const struct shell_transport *transport,
 		 const void *data, size_t length, size_t *cnt)
 {
 	struct shell_dummy *sh_dummy = (struct shell_dummy *)transport->ctx;
+	size_t store_cnt;
 
 	if (!sh_dummy->initialized) {
 		*cnt = 0;
 		return -ENODEV;
 	}
 
+	store_cnt = length;
+	if (sh_dummy->ptr + store_cnt >= sizeof(sh_dummy->buf))
```

Comment:
brackets are always required according to the coding standard.

Reply:
I only see this https://docs.zephyrproject.org/latest/contribute/index.html#coding-style and the reference to kernel style. Can you point me to the reference for adding extra brackets?

I use patman (which runs checkpatch) to check the patches.

Reply:
OK I misunderstood this, it is braces

Reply:
right, i meant braces. From the link you pasted:
```
In general, follow the Linux kernel coding style, with the following exceptions:

Add braces to every if, else, do, while, for and switch body, even for single-line code blocks.
```